### PR TITLE
[GAIA CI Tests] Unify gremlin ci tests under different storages

### LINKED ISSUE #2510: [GIE CI Tests] Unify gremlin CI tests under different storages (expr/groot/vineyard)
**Describe the solution you'd like**

Currently, GIE has an independent entry-point (launch service and test suites) to run tests for each different storages (expr/groot/vineyard), which is very inconvenient for adding and deleting tests when necessary. 

To solve the problem, test suites should be unified by taking a subset of the three test sets firstly. Then, GIE can has a unified test entry-point after the service launching has also been unified.

### PR DESCRIPTION
**Pull request by shirly121**
## What do these changes do?
1. unify gremlin ci tests under different storages (groot/expr/vineyard), see [RemoteTestGraph.java](https://github.com/shirly121/GraphScope/blob/ir_integrate_ci_tests/interactive_engine/compiler/src/main/java/com/alibaba/graphscope/gremlin/integration/graph/RemoteTestGraph.java).
2. reorg test suite package from `src/test/java/com/alibaba/graphscope/integration` to `src/main/java/com/alibaba/graphscope/gremlin/integration/suite` , to be imported by other modules.

## Related issue number

Fixes #2510

### CODE REVIEW

**Review by siyuan0322 — APPROVED** [review]

**Review by longbinlai — COMMENTED** [review]
Can we provide a table of test cases that are removed due to at least one store does not support the cases, including information includes: 

test_cases, unsupported store [vineyard|groot|...], why it is not supported

<html xmlns:o="urn:schemas-microsoft-com:office:office"
xmlns:w="urn:schemas-microsoft-com:office:word"
xmlns:m="http://schemas.microsoft.com/office/2004/12/omml"
xmlns="http://www.w3.org/TR/REC-html40">

<head>

<meta name=ProgId content=Word.Document>
<meta name=Generator content="Microsoft Word 15">
<meta name=Originator content="Microsoft Word 15">
<link rel=File-List
href="file:////Users/zhouxiaoli/Library/Group%20Containers/UBF8T346G9.Office/TemporaryItems/msohtmlclip/clip_filelist.xml">

<link rel=themeData
href="file:////Users/zhouxiaoli/Library/Group%20Containers/UBF8T346G9.Office/TemporaryItems/msohtmlclip/clip_themedata.thmx">
<link rel=colorSchemeMapping
href="file:////Users/zhouxiaoli/Library/Group%20Containers/UBF8T346G9.Office/TemporaryItems/msohtmlclip/clip_colorschememapping.xml">

<style>

</style>

</head>

<body lang=ZH-CN style='tab-interval:21.0pt;word-wrap:break-word;text-justify-trim:
punctuation'>


<p class=MsoNormal align=left style='text-align:left'><span lang=EN-US
style='font-size:12.0pt;font-family:"Times New Roman",serif'><o:p>&nbsp;</o:p></span></p>



Test Case | Unsupported Storages | Reasons
-- | -- | --
g_V_hasXp_neqXvXX   g_V_hasXblahX 
g_VX1X_hasXcircumferenceX | Vineyard & Groot | Ir core will convert   a non-existing property name to a magic number which is negative, runtime   will throw exceptions if the given property id is negative.
g.E(XX) | Vineyard &   Groot | Fetching edge by   its inner id is unsupported in vineyard or groot



<p class=MsoNormal align=left style='text-align:left'><span lang=EN-US
style='font-size:12.0pt;font-family:"Times New Roman",serif'><o:p>&nbsp;</o:p></span></p>


</body>

</html>

**Comment by shirly121** [comment]
> Can we provide a table of test cases that are removed due to at least one store does not support the cases, including information includes:
> 
> test_cases, unsupported store [vineyard|groot|...], why it is not supported

<html xmlns:o="urn:schemas-microsoft-com:office:office"
xmlns:w="urn:schemas-microsoft-com:office:word"
xmlns:m="http://schemas.microsoft.com/office/2004/12/omml"
xmlns="http://www.w3.org/TR/REC-html40">

<head>

<meta name=ProgId content=Word.Document>
<meta name=Generator content="Microsoft Word 15">
<meta name=Originator content="Microsoft Word 15">
<link rel=File-List
href="file:////Users/zhouxiaoli/Library/Group%20Containers/UBF8T346G9.Office/TemporaryItems/msohtmlclip/clip_filelist.xml">

<link rel=themeData
href="file:////Users/zhouxiaoli/Library/Group%20Containers/UBF8T346G9.Office/TemporaryItems/msohtmlclip/clip_themedata.thmx">
<link rel=colorSchemeMapping
href="file:////Users/zhouxiaoli/Library/Group%20Containers/UBF8T346G9.Office/TemporaryItems/msohtmlclip/clip_colorschememapping.xml">

<style>

</style>

</head>

<body lang=ZH-CN style='tab-interval:21.0pt;word-wrap:break-word;text-justify-trim:
punctuation'>




Test Case | Unsupported Storages | Reasons
-- | -- | --
g_V_hasXp_neqXvXX   g_V_hasXblahX g_VX1X_hasXcircumferenceX | Vineyard & Groot | Ir core will convert   a non-existing property name to a magic number which is negative, runtime   will throw exceptions if the given property id is negative.
g.E(XX) | Vineyard &   Groot | Fetching edge by   its inner id is unsupported in vineyard or groot




</body>

</html>

File: .github/workflows/gss.yml [hunk 1]
```diff
@@ -76,6 +76,7 @@ jobs:
       run: |
         source ${HOME}/.graphscope_env
         cd interactive_engine/groot-server
+        sleep 1000000
```
Comment (yecol) [inline .github/workflows/gss.yml:79]: Maybe need to be removed?

Comment (shirly121) [reply .github/workflows/gss.yml]: just add it for debug, have removed it.

**Pull request merged**